centreon/www/front_src/src/Resources/Filter/SearchHelp.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+
+import { useAtomValue } from 'jotai';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from 'tss-react/mui';
+
+import { Link, Box, Typography } from '@mui/material';
+
+import { PersistentTooltip } from '@centreon/ui';
+
+import { platformVersionsAtom } from '../../Main/atoms/platformVersionsAtom';
+import {
+  labelFindExplanationsAndExamples,
+  labelHere,
+  labelNeedHelpWithSearchBarUsage,
+  labelSearchHelp
+} from '../translatedLabels';
+
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    color: theme.palette.common.black,
+    margin: theme.spacing(1, 0),
+    marginRight: theme.spacing(3)
+  },
+  link: {
+    color: theme.palette.primary.main
+  },
+  title: {
+    marginBottom: theme.spacing(0.5)
+  }
+}));
+
+const SearchHelp = (): JSX.Element => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+
+  const [openTooltip, setOpenTooltip] = useState(false);
+  const platform = useAtomValue(platformVersionsAtom);
+
+  const docsURL = `https://docs.centreon.com/docs/${platform?.web.major}.${platform?.web.minor}/alerts-notifications/resources-status/#search-bar`;
```

Comment:
the link anchor in French is different

Reply:
In the latest version of the documentation (currently 23.04), URLs do not include a version number. Therefore, a link like this one: https://docs.centreon.com/fr/docs/23.04/alerts-notifications/resources-status/#search-bar does not work (page not found). However, if we use https://docs.centreon.com/docs/23.04/alerts-notifications/resources-status/#search-bar, it will be redirected to https://docs.centreon.com/docs/alerts-notifications/resources-status/#search-bar.

On our side, we do not know if it is the latest version to handle this case.